src/app/_services/auth-api.ts

```diff
@@ -0,0 +1,43 @@
+import { LoginFormType } from "../_components/Login";
+import { RegisterFormType } from "../_components/Register";
+import { ResponseLogin, User } from "../_utils/type-common";
+import { post } from "./axiosConfig";
+
+export async function registerApi({
+  stringUrl,
+  data,
+}: {
+  stringUrl: string;
+  data: RegisterFormType;
+}) {
+  await post({
+    url: `/Auth/${stringUrl}`,
```

Comment:
```suggestion
    url: "/Auth/register",
```
TƯơng tự mấy chỗ dưới